lib/emu/emulator-state.spec.ts

```diff
@@ -102,7 +102,8 @@ describe('The EmulatorState - handle state changes', () => {
 	});
 
 	it('should return empty array for initially unchanged lamps', () => {
-		expect(emulatorState.getChangedLamps()).to.deep.equal([]);
+		expect(emulatorState.getChangedLamps()).to.be.an('array');
+		expect(emulatorState.getChangedLamps()).to.be.empty;
```

Comment:
more a personal preference - but i really like the simple tap test interface (basically equal) vs. the ton's of specific test framework specific test functions.
as i said current implementation works fine, there is nothing wrong - but I think

```
expect(typeof emulatorState.getChangedLamps()).to.equal('array');
expect(emulatorState.getChangedLamps().length).to.equal(0);
```

is easier to read and maintain

Reply:
Well, it's Chai, so I usually have the reference open when *writing* tests. But *reading*? The whole goal of the Chai test functions are readability, no? Personally I like them, specially when reading.